### ISSUE TITLE
Update Serbian translations

### ISSUE THREAD

**Issue by stcksmsh** [squidfunk/mkdocs-material#6999]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "sr",
  "action.edit": "Ажурирај страницу",
  "action.skip": "Иди на текст",
  "action.view": "Погледајте изворни код ове странице",
  "announce.dismiss": "Не приказуј поново",
  "blog.archive": "Архива",
  "blog.categories": "Категорије",
  "blog.categories.in": "у",
  "blog.continue": "Настави са читањем",
  "blog.draft": "Нацрт",
  "blog.index": "Назад на индекс",
  "blog.meta": "Метаподаци",
  "blog.references": "Сродне везе",
  "clipboard.copy": "Копирај у клипборд",
  "clipboard.copied": "Ископирано у клипборд",
  "consent.accept": "Прихвати",
  "consent.manage": "Управљај подешавањима",
  "consent.reject": "Одбиј",
  "footer": "Подножје",
  "footer.next": "Следеће",
  "footer.previous": "Претходно",
  "header": "Заглавље",
  "meta.comments": "Коментари",
  "meta.source": "Извор",
  "nav": "Навигација",
  "readtime.one": "1 минут читања",
  "readtime.other": "# минута читања",
  "rss.created": "РСС феед",
  "rss.updated": "РСС феед ажурираног садржаја",
  "search": "Претрага",
  "search.placeholder": "Претрага",
  "search.share": "Дељење",
  "search.reset": "Очисти",
  "search.result.initializer": "Иницијализујем претрагу",
  "search.result.placeholder": "Унесите појам претраге",
  "search.result.none": "Ништа није пронађено",
  "search.result.one": "1 резултат претраге",
  "search.result.other": "# резултата претраге",
  "search.result.more.one": "још 1 на овој страни",
  "search.result.more.other": "још # на овој страни",
  "search.result.term.missing": "Недостаје",
  "select.language": "Изабери језик",
  "select.version": "Изабери верзију",
  "source": "Иди у репозиторијум",
  "source.file.contributors": "Доприноситељи",
  "source.file.date.created": "Креиран",
  "source.file.date.updated": "Ажуриран",
  "tabs": "Табови",
  "toc": "Садржај",
  "top": "Назад на врх"
}[key] }}{% endmacro %}
```

### Country flag

:flag_rs:

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks for providing the missing translations! Added in a3655e830.

**Comment by squidfunk** [comment]
Released as part of [9.5.17](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.17).
